numpy/core/arrayprint.py

```diff
@@ -708,9 +708,9 @@ def __call__(self, x):
             i = i + 'j'
         return r + i
 
+
 class DatetimeFormat(object):
-    def __init__(self, x, unit=None,
-                timezone=None, casting='same_kind'):
+    def __init__(self, x, unit=None, timezone='naive', casting='same_kind'):
```

Comment:
The lack of a naive time zone was probably the biggest problem in the datetime NEP as it was when I started doing any datetime work. Unfortunately I didn't realize it should be added, and the discussion at the time didn't help with that.